package.json

```
@@ -10,6 +10,9 @@
     "url": "https://github.com/JedWatson/classnames.git"
   },
   "types": "./index.d.ts",
+  "engines": {
+    "node": ">=12.22.0"
```

```suggestion
    "node": "^12.22 || >= 14.17"
```

As per your suggestion in https://github.com/JedWatson/classnames/pull/243#discussion_r635640583,  we can drop that to `12.20` and `14.13` if that we want